File: re-ssim-components/re-ssim-data-manager-application/src/main/java/com/shinonometn/re/ssim/data/manager/application/repository/ImportTaskRepository.java
package com.shinonometn.re.ssim.data.manager.application.repository;

import com.shinonometn.re.ssim.service.data.ImportTask;
import org.springframework.data.mongodb.repository.MongoRepository;

public interface ImportTaskRepository extends MongoRepository<ImportTask,String>{

    boolean existsByCaptureTaskId(String captureTaskId);
}

File: re-ssim-components/re-ssim-data-kingo-application/src/main/java/com/shinonometn/re/ssim/data/kingo/application/courses/repository/ImportTaskRepository.kt
package com.shinonometn.re.ssim.data.kingo.application.courses.repository

import com.shinonometn.re.ssim.data.kingo.application.courses.entity.ImportTask
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.mongodb.repository.MongoRepository

interface ImportTaskRepository : JpaRepository<ImportTask, Int> {

    fun existsByCaptureTaskId(captureTaskId: Int?): Boolean
}
